lib/wordsmith/client.rb

```diff
@@ -6,49 +6,55 @@ module Wordsmith
   class Client
 
     def get(uri)
-      response = connection.get uri
-      parse_response response
+      response = connection.get(uri)
+      parse_response(response)
     end
 
-    def post(uri, data)
-      response = connection.post uri, {data: data}.to_json
-      parse_response response
+    def post(uri, data, proofread)
```

Comment:
I'm not a fan of flag parameters like this. If you ARE going to use it, I would suggest having it as a keyword arg & setting a default here (`proofread: false`) which you're already doing in the `Template` class. Since the flag is part of the API itself as a flag* I think it's fine as long as you use it as a keyword arg here. 

* I didn't see anything about this in the [API docs](http://wordsmith.readme.io/v1/docs)...do they need to get updated?